app/build.gradle

```diff
@@ -234,11 +234,15 @@ dependencies {
     compile "com.squareup.retrofit2:converter-gson:$versions.retrofitVersion"
     compile "com.squareup.okhttp3:logging-interceptor:$versions.loggingInterceptorVersion"
 
+
     // Rx
-    compile "io.reactivex:rxandroid:$versions.rxAndroidVersion"
+    compile "io.reactivex.rxjava2:rxandroid:$versions.rxAndroidVersion" //rxjava2
     // Already has RxJava, but still better to explicitly get latest version.
-    compile "io.reactivex:rxjava:$versions.rxJavaVersion"
-    compile "com.squareup.retrofit2:adapter-rxjava:$versions.retrofitVersion"
+    compile "io.reactivex.rxjava2:rxjava:$versions.rxJavaVersion"  //rxjava2
+    compile "com.squareup.retrofit2:adapter-rxjava2:$versions.retrofitVersion" //rxjava
+    //0.16.0" //for assemblyTracking in debug
```

Comment:
can you please remove the '' //0.16.0"  part of the comment.
Also, do you happen to have any links on whether they are planning to add this feature back into the official library or why they removed it in the first place?